src/Graphics/Glyphs/GlyphGeom.cc

```diff
@@ -311,29 +316,29 @@ void GlyphGeom::generateCylinder(const Point& p1, const Point& p2, double radius
   Vector p;
   for (int strips = 0; strips <= num_strips; strips++)
   {
-    size_t offset = static_cast<size_t>(numVBOElements_);
+    size_t offset = static_cast<size_t>(geomData_.numVBOElements);
     p = std::cos(2. * M_PI * strips / num_strips) * u +
       std::sin(2. * M_PI * strips / num_strips) * crx;
     p.normalize();
     Vector normals(((p2-p1).length() * p + (r2-r1)*n).normal());
 
-    points_.push_back(r1 * p + Vector(p1));
-    colors_.push_back(color1);
-    normals_.push_back(normals);
-    numVBOElements_++;
-    points_.push_back(r2 * p + Vector(p2));
-    colors_.push_back(color2);
-    normals_.push_back(normals);
-    numVBOElements_++;
-
-    indices_.push_back(0 + offset);
-    indices_.push_back(1 + offset);
-    indices_.push_back(2 + offset);
-    indices_.push_back(2 + offset);
-    indices_.push_back(1 + offset);
-    indices_.push_back(3 + offset);
+    geomData_.points.push_back(r1 * p + Vector(p1));
```

Comment:
These four lines also make a method--it's the same 4 modifications.

Reply:
There's now a function addIndices() which takes 3 index values to form a triangle. Also that offset value can be set and is held internally now.